Dockerfile.tools

```
@@ -1,24 +1,22 @@
 FROM registry.svc.ci.openshift.org/ocp/builder:rhel-8-golang-1.15-openshift-4.6 as build
```

@aravindhp should I bump this to 4.7?
```suggestion
FROM registry.svc.ci.openshift.org/ocp/builder:rhel-8-golang-1.15-openshift-4.7 as build
```

---

Yes please given this is going into master.